cw-orch-daemon/examples/hd_index.rs

```diff
@@ -0,0 +1,26 @@
+use cosmwasm_std::coins;
+use cw_orch::{anyhow, prelude::*};
+use cw_orch_daemon::CosmosOptions;
+
+// From https://github.com/CosmosContracts/juno/blob/32568dba828ff7783aea8cb5bb4b8b5832888255/docker/test-user.env#L1
+pub const LOCAL_JUNO_SENDER: &str = "juno16g2rahf5846rxzp3fwlswy08fz8ccuwk03k57y";
+pub const LOCAL_JUNO_GRANTER: &str = "juno1afhtjur8js4589xymu346ca7a5y5293xpuv6ry";
+
+pub fn main() -> anyhow::Result<()> {
+    pretty_env_logger::init(); // Used to log contract and chain interactions
+
+    let network = networks::LOCAL_JUNO;
+    // There is no need to register a mnemonic to use this daemon querier
```

Comment:
Was is meant here with "no need to register a mnemonic"? 

Do you mean that the mnemonic is read from the env variables?